migrations/Version202107161250081888_taoMediaManager.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\taoMediaManager\migrations;
+
+use oat\taoMediaManager\controller\MediaManager;
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\taoMediaManager\model\classes\user\TaoAssetRoles;
+use oat\tao\scripts\tools\accessControl\SetRolesAccess;
+use oat\tao\scripts\update\OntologyUpdater;
+use oat\tao\model\accessControl\ActionAccessControl;
+
+final class Version202107161250081888_taoMediaManager extends AbstractMigration
+{
+    private const CONFIG = [
+        SetRolesAccess::CONFIG_RULES => [
+            TaoAssetRoles::ASSET_CLASS_NAVIGATOR => [
+                ['ext' => 'taoMediaManager', 'mod' => 'MediaManager', 'act' => 'editClassLabel'],
+                ['ext' => 'taoMediaManager', 'mod' => 'MediaManager', 'act' => 'index'],
+                ['ext' => 'taoMediaManager', 'mod' => 'MediaManager', 'act' => 'getOntologyData']
+            ],
+            TaoAssetRoles::ASSET_VIEWER => [
+                ['ext' => 'taoMediaManager', 'mod' => 'MediaManager', 'act' => 'editInstance'],
+                ['ext' => 'taoMediaManager', 'mod' => 'SharedStimulus', 'act' => 'get'],
+            ],
+            TaoAssetRoles::ASSET_EXPORTER => [
+                ['ext' => 'taoMediaManager', 'mod' => 'MediaExport', 'act' => 'index'],
+            ],
+            TaoAssetRoles::ASSET_PREVIEWER => [
+                ['ext' => 'taoMediaManager', 'mod' => 'MediaManager', 'act' => 'editInstance'],
+                ['ext' => 'taoMediaManager', 'mod' => 'MediaManager', 'act' => 'getFile'],
+            ],
+            TaoAssetRoles::ASSET_CONTENT_CREATOR => [
+                ['ext' => 'taoMediaManager', 'mod' => 'MediaManager', 'act' => 'authoring'],
+                ['ext' => 'taoMediaManager', 'mod' => 'MediaImport', 'act' => 'editMedia'],
+                ['ext' => 'taoMediaManager', 'mod' => 'MediaManager', 'act' => 'editInstance'],
+                ['ext' => 'taoMediaManager', 'mod' => 'SharedStimulus', 'act' => 'patch'],
+            ],
+        ],
+        SetRolesAccess::CONFIG_PERMISSIONS => [
+            MediaManager::class => [
+                'editClassLabel' => [
+                    TaoAssetRoles::ASSET_CLASS_NAVIGATOR => ActionAccessControl::READ
+                ],
+                'editInstance' => [
+                    TaoAssetRoles::ASSET_VIEWER => ActionAccessControl::READ,
+                    TaoAssetRoles::ASSET_PREVIEWER => ActionAccessControl::READ,
+                    TaoAssetRoles::ASSET_PROPERTIES_EDITOR => ActionAccessControl::WRITE,
+                    TaoAssetRoles::ASSET_CONTENT_CREATOR => ActionAccessControl::READ,
```

Comment:
If I'm not mistaken, `ASSET_VIEWER` and `ASSET_PROPERTIES_EDITOR` will be enough.
Because `ASSET_PREVIEWER` and `ASSET_CONTENT_CREATOR` includes `ASSET_VIEWER` role.
Can you check this, please?

```suggestion
                    TaoAssetRoles::ASSET_VIEWER => ActionAccessControl::READ,
                    TaoAssetRoles::ASSET_PROPERTIES_EDITOR => ActionAccessControl::WRITE,
```

Reply:
Unfortunately they do not extend ASSET_VIEWER @shpran :/

https://oat-sa.atlassian.net/browse/ADF-208

Reply:
@gabrielfs7 

| # | Role | Includes |
| -- | -- | -- |
| `AR1` | Asset viewer | ACR1 |
| AR3 | Asset previewer | AR1 |
| AR4 | Asset properties editor | `AR1` |
| AR7 | Asset content creator | `AR1` |

Reply:
Yes, you are completely right. I was blind, but now I see :)

Reply:
Done